github/resource_github_branch_protection.go

```diff
@@ -297,11 +318,9 @@ func flattenRequiredPullRequestReviews(d *schema.ResourceData, protection *githu
 		}); err != nil {
 			return err
 		}
-	} else {
-		d.Set("required_pull_request_reviews", []interface{}{})
 	}
 
-	return nil
+	return d.Set("required_pull_request_reviews", []interface{}{})
 }
 
 func flattenRestrictions(d *schema.ResourceData, protection *github.Protection) error {
```

Comment:
(same here)

Reply:
Yeah, I'll change it. I kind of wish the function wasn't as greedy and did one thing only, but that's refactoring for another time.